templates/web/base/report/new/category.html

```diff
@@ -5,7 +5,11 @@
     <label for='form_category' id="form_category_label">
         [%~ loc('Category') ~%]
     </label>[% =%]
-    <select class="form-control" name='category' id='form_category' data-role='[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]' data-prefill="[% c.cobrand.prefill_report_fields_for_inspector %]">
+    <select class="form-control" name="category" id="form_category"
+    [%~ IF c.user.from_body ~%]
+      data-role="[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]" data-body="[% c.user.from_body.name %]" data-prefill="[% c.cobrand.prefill_report_fields_for_inspector %]"
+    [%~ END ~%]
+    required>
```

Comment:
This wasn't `required` before - not sure if it should be or not, but probably shouldn't be done as an aside to this change.